index.bs

```diff
@@ -546,28 +568,93 @@ canvas.height = Math.max(leftEye.renderHeight, rightEye.renderHeight);
+
+This frame of reference's roll and pitch are relative to gravity while the initial yaw and origin position were based on the {{VRDisplay}} from which the frame of reference was created.
+
+As this frame of reference may become untrackable due to environmental changes, developers are encouraged to be prepared fall back to body-locked content rendered using a {{VRAttachedFrameOfReference}}.
+
+<pre class="idl">
+interface VRStationaryFrameOfReference {
+  readonly attribute VRCoordinateSystem coordinateSystem;
+};
+</pre>
+
+### Attributes ### {#vrstationaryframeofreference-attributes}
+
+<dfn attribute for="VRStationaryFrameOfReference">coordinateSystem</dfn>
+This attribute is the {{VRCoordinateSystem}} to be used when relating objects from this stationary frame of reference to objects in other coodinate systems.
+
+
+## VRStageFrameOfReference ## {#interface-vrstageframeofreference}
+
+The {{VRStageFrameOfReference}} interface represents the stage/play area for devices that support room-scale experiences.  The values within {{VRStageFrameOfReference}} may change after any rendering frame is submitted as the user may re-configure their environment at any time.
+
+Room-scale experiences are expected to use the VRStageFrameOfReference rather than the VRStationaryFrameOfReference, otherwise they may experience drift on world-scale devices.
+
+<pre class="idl">
+interface VRStageFrameOfReference {
+  readonly attribute VRCoordinateSystem coordinateSystem;
 
   readonly attribute float sizeX;
   readonly attribute float sizeZ;
 };
 </pre>
 
-### Attributes ### {#vrstageparameters-attributes}
+### Attributes ### {#vrstageframeofreference-attributes}
 
-<dfn attribute for="VRStageParameters">sittingToStandingTransform</dfn>
-The {{sittingToStandingTransform}} attribute is a 16-element array containing the components of a 4x4 affine transformation matrix in column-major order. This matrix transforms the sitting-space view matrices of {{VRFrameData}} to standing-space. Multiplying the inverse of this matrix with the {{leftViewMatrix}} or {{rightViewMatrix}} will result in a standing space view matrix for the respective eye.
+<dfn attribute for="VRStageFrameOfReference">coordinateSystem</dfn>
+This attribute is the {{VRCoordinateSystem}} to be used when relating objects in this stage to objects in other coordinate systems.
 
-<dfn attribute for="VRStageParameters">sizeX</dfn>
+<dfn attribute for="VRStageFrameOfReference">sizeX</dfn>
 Width of the play-area bounds in meters. The bounds are defined as an axis-aligned rectangle on the floor. The center of the rectangle is at (0,0,0) in standing-space coordinates. These bounds are defined for safety purposes. Content should not require the user to move beyond these bounds; however, it is possible for the user to ignore the bounds resulting in position values outside of this rectangle.
```

Comment:
While we're here can we add a note that this may be zero if the bounds aren't known?